apps/coscrad-frontend-e2e/src/e2e/categories/category-tree.query-flow.e2e.cy.ts

```diff
@@ -0,0 +1,65 @@
+describe('Categories Tree (Tree of Knowledge) flow', () => {
+    const pageLabel = 'Tree of Knowledge';
+
+    const dummyCompositeIdPrefix = 'category/9b1deb4d-3b7d-4bad-9bdd-2b0d7b11000';
```

Comment:
There is tooling for generating dummy composite IDs. Please use `buildDummyAggregateCompositeIdentifier` along with a custom format function.

Reply:
This is temporary, I think I tried dummy generation and got an extra `1`